packages/protocol/contracts/ACE/ACE.sol

```diff
@@ -297,107 +441,27 @@ contract ACE is IAZTEC {
         updateOutputNotes(outputNotes);
 
         if (publicValue != 0) {
-            require(registry.flags.canMint == false, "mintable assets cannot be converted into public tokens");
-            require(registry.flags.canBurn == false, "burnable assets cannot be converted into public tokens");
             require(registry.flags.canConvert == true, "this asset cannot be converted into public tokens");
+
             if (publicValue < 0) {
                 registry.totalSupply = registry.totalSupply.add(uint256(-publicValue));
                 require(
                     registry.publicApprovals[publicOwner][proofHash] >= uint256(-publicValue),
                     "public owner has not validated a transfer of tokens"
                 );
+                 
                 registry.publicApprovals[publicOwner][proofHash] -= uint256(-publicValue);
                 require(
                     registry.linkedToken.transferFrom(publicOwner, address(this), uint256(-publicValue)), 
                     "transfer failed"
                 );
-            } else {
-                registry.totalSupply = registry.totalSupply.sub(uint256(publicValue));
+            } else { 
+                registry.totalSupply < uint256(publicValue);
```

Comment:
I think this is a typo? We absolutely need SafeMath.sub here to protect against a user stealing tokens out of another note registry

Reply:
Very good catch. This is an out of place line of code, left over from when I was previously adapting the updateNoteRegistry() function. 

I've replaced it with the correct intended line, surprised it compiled